opa/Dockerfile

```
@@ -71,10 +73,10 @@ COPY opa/licenses /licenses
 USER stackable
 WORKDIR /stackable/opa
 
-RUN curl --fail -L https://repo.stackable.tech/repository/packages/opa/opa_linux_amd64_static_${PRODUCT} \
-     -o opa_linux_amd64_static_${PRODUCT} && \
-    chmod +x opa_linux_amd64_static_${PRODUCT} && \
-    ln -s opa_linux_amd64_static_${PRODUCT} opa
+RUN curl --fail -L "https://repo.stackable.tech/repository/packages/opa/opa_${TARGETOS}_${TARGETARCH}_static_${PRODUCT}" \
+     -o opa_linux_${ARCH}_static_${PRODUCT} && \
+    chmod +x opa_linux_${ARCH}_static_${PRODUCT} && \
+    ln -s opa_linux_${ARCH}_static_${PRODUCT} opa
```

`${ARCH}` must be replaced with `${TARGETARCH}`:

```
bash-4.4$ ls -l
total 39332
lrwxrwxrwx 1 stackable stackable       24 Feb  9 11:07 opa -> opa_linux__static_0.61.0
-rwxr-xr-x 1 stackable stackable 40269280 Feb  9 11:07 opa_linux__static_0.61.0
```

```suggestion
     -o opa_${TARGETOS}_${TARGETARCH}_static_${PRODUCT} && \
    chmod +x opa_${TARGETOS}_${TARGETARCH}_static_${PRODUCT} && \
    ln -s opa_${TARGETOS}_${TARGETARCH}_static_${PRODUCT} opa
```

---

done with https://github.com/stackabletech/docker-images/pull/534/commits/272e4e1ac9491302257e2e03077ea619edfae7e2